scripts/mkdata.py

```diff
@@ -105,125 +65,200 @@ def gen_r_script(data, layout, single_eas, out_dir):
      at = seq(1 , 5 , 1),
       labels = c('(a)','(b)','(c)','(d)','?'),
      tick=T , cex=0.3)
-abline(v=2.5,lty=1, col="grey")
+abline(v=${line}.5,lty=1, col="grey")
 
-lvl <- levels(data$sample)
-props <- summary(data$sample) / nrow(data)
+lvl <- levels(data$$sample)
+props <- summary(data$$sample) / nrow(data)
 
 for (i in 1:length(lvl)) {{
     l <- lvl[i]
-    v <- data[ data$sample==l, "values" ]
+    v <- data[ data$$sample==l, "values" ]
     j <- jitter(rep(i, length(v)), amount=props[i]/2)
     points(j, v, pch=20, col=rgb(0,0,0,0.7))
 }}
 """
 
-    script += f"# Total n: {ns} (should be 1024)\n"
-    script += "dev.off()\n"
-    out_dir.mkdir(parents=True, exist_ok=True)
-    with open(out_dir / "plot.R", "w") as stream:
-        stream.write(script)
+R_SCRIPT_FOOTER_TEMPLATE = """
+# Total n: ${ns} (should be 1024)
```

Comment:
Still 1024? This was a debug info for me. If this isn't true anymore, you can just remove the "should be part".